ldn-service.js

```diff
@@ -200,24 +221,33 @@ function factory($http, $q) {
 
     options = angular.extend({
       index: self.messages.length,
-      count: Math.min(self.getMessages.count, self.messages.length)
+      count: Math.min(
+        self.defaults.getMessages.count, self.container.contains.length)
     }, options || {});
 
     var start = options.index;
-    var slice = self.container.slice(start, start + options.count);
+    var slice = self.container.contains.slice(start, start + options.count);
 
     return $q.all(slice.map(function(url, idx) {
+      // FIXME: Remove if not needed
+      // return jsonld.promises.compact(url, CONTEXT_URL_AS)
+      //   .then(function(compacted) {
+      //     console.log('HHHHHHHHH', compacted);
+      //   });
       // TODO: do JSON-LD compaction to CONTEXT_URL
```

Comment:
Since we're doing framing below we shouldn't need to do any compaction (framing does this automatically). Can we remove the FIXME and the TODO?